cwizard.py

```diff
@@ -0,0 +1,124 @@
+"""Module to help the user build a dictionary and choose options"""
+# Author: Daniel Davis

```

Comment:
Can you remove these 3 comments? In the end, it's just more code we will have to maintain (on CleanCode there is a chapter explaining this in more detail). Basically GIT will do this for us and make sure we know who were the authors, when and etc

Reply:
It was always a requirement on every project I worked on to sign your code. If you have a different way of working, then that is something which needs to be discussed and agreed as a team. We have five people with very different backgrounds - and that is good, but inevitably we all have our own experiences and ways of doing things. Perhaps we need as a team discuss the project management methods that we are following.